lms/static/my.js

```diff
@@ -106,14 +106,39 @@ function trackReadAllNotificationsButton(button) {
   });
 }
 
+function getPostUploadMessage() {
+  const myDropzone = Dropzone.forElement('#demo-upload');
+  const feedbacks = document.getElementById('upload-feedbacks');
+  const matchesSpan = document.getElementById('upload-matches');
+  const missesSpan = document.getElementById('upload-misses');
+  myDropzone.on('success', function() {
+    const args = Array.from(arguments).slice(1)[0];
+    const matches = args['exercise_matches'];
+    const misses = args['exercise_misses'];
+    if (matches.length) {
+      matchesSpan.innerHTML += matches + ',';
+    }
+    if (misses.length) {
+      missesSpan.innerHTML += misses + ',';
+    }
+    feedbacks.classList.remove('d-none');
```

Comment:
Maybe we should add some kind of transition in the CSS when the content is changed? :)